test/functional/JLM_Tests/variables.mk

```diff
@@ -0,0 +1,30 @@
+##############################################################################
+#  Copyright (c) 2022, 2022 IBM Corp. and others
+#
+#  This program and the accompanying materials are made available under
+#  the terms of the Eclipse Public License 2.0 which accompanies this
+#  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+#  or the Apache License, Version 2.0 which accompanies this distribution and
+#  is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+#  This Source Code may also be made available under the following
+#  Secondary Licenses when the conditions for such availability set
+#  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+#  General Public License, version 2 with the GNU Classpath
+#  Exception [1] and GNU General Public License, version 2 with the
+#  OpenJDK Assembly Exception [2].
+#
+#  [1] https://www.gnu.org/software/classpath/license.html
+#  [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+#  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+##############################################################################
+
+# In JDK18+, java.security.manager == null behaves as -Djava.security.manager=disallow.
+# In JDK17-, java.security.manager == null behaves as -Djava.security.manager=allow.
+# For OpenJ9 tests to work as expected, -Djava.security.manager=allow behaviour is
+# needed in JDK18+.
+JAVA_SECURITY_MANAGER=
+ifeq ($(filter 8 9 10 11 12 13 14 15 16 17, $(JDK_VERSION)),)
+ JAVA_SECURITY_MANAGER=-Djava.security.manager=allow
+endif
```

Comment:
I think it would be better if we have only one variables.mk in test/functional directory and all the playlist.xml files include this one. It would be easier to maintain later on.

Reply:
Cool, that will reduce duplicate code. Can you confirm the below change to derive the absolute path?
```
<include>$(Q)$(JVM_TEST_ROOT)$(D)functional$(D)global_variables.mk$(Q)</include>
```

Reply:
The below code does not work:
```
	<include>$(Q)$(JVM_TEST_ROOT)$(D)functional$(D)variables.mk$(Q)</include>
```
Only relative paths are accepted:
```
	<include>../../variables.mk</include>
```
Also, multiple includes do not work. e.g. `test/functional/Java8andUp/playlist.xml`:
```
	<include>java8andUpSettings.mk</include>
	<include>../variables.mk</include>
```
@renfeiw is it possible to include multiple makefiles and allow absolute paths for the makefiles?

Reply:
this should work:
```
<include>$(TEST_ROOT)$(D)functional$(D)variables.mk</include>
```
Currently TKG only takes single <include>, I can add support for multiple later on. A workaround for anything already have a makefile is to include the functional makefile in the existing sub level makefile.

Reply:
Discussing the `inlcude` approach with @renfeiw.

Reply:
Updated.